scripts/normalize-jest-coverage.js

```diff
@@ -0,0 +1,29 @@
+const libCoverage = require('istanbul-lib-coverage');
```

Comment:
hopefully we can remove this and use codecov's native report merging -- will see.

Reply:
Update: attempted to get codecov to natively detect and merge multiple test coverage files without success. Punting on further progress on this front for now to prioritize other initiatives.